charts/pwa/values.yaml

```diff
@@ -188,29 +188,5 @@ cache:
   readinessProbe: {}
 
 ingress:
-  enabled: true
-  className: nginx
-  annotations: {}
-  hosts:
-    - host: pwa.example.local
-      paths:
-        - path: /
-          pathType: ImplementationSpecific
-  tls: []
-  #  - secretName: chart-example-tls
-  #    hosts:
-  #      - chart-example.local
-
-ingresssplit:
   enabled: false
   className: nginx
```

Comment:
The new convenient format does not allow the creation of a default ingress, because it would always create it and this will cause issues for operations. But this feature will probably only be used for development anyways.

Reply:
As long as it is possible to still create one single ingress, for everyday usage or customers with only 1 channel or few channels that are all live, then i see no problem in that.

Reply:
Yes of course, whatever you declare will get created. It just now doesn't create anything by default if you leave it empty.

So if you just deploy the chart without a values file, you won't get an ingress anymore.
It will still print out some information about forwarding the port to localhost.